src/Compilers/CSharp/Test/Symbol/Symbols/Source/DeclaringSyntaxNodeTests.cs

```diff
@@ -421,28 +464,27 @@ namespace System {}
         }
 
         [Fact]
-        public void TypeParameterDeclaringSyntax()
+        public void TypeParameterDeclaringSyntaxSingleLineNamespace()
```

Comment:
I didn't understand why the "regular" namespace version of the test was essentially removed here.

Reply:
Good point. Let's add the original back, or make this a Theory to cover both